crates/relayer-framework-test/src/relayer_mock/contexts/chain.rs

```diff
@@ -355,8 +360,11 @@ impl MockChainContext {
         )
     }
 
-    /// Update the chain's current_state and past_chain_states at the same time to insure
-    /// they are always synchronized.
+    /// Update the chain's current_state and past_chain_states at the same time to ensure
+    /// they are always synchronized. Accepts an optional `duration` parameter that is
```

Comment:
Is the optional `duration` missing in the method's parameters ?

Reply:
I had changed the function's signature in a previous commit and updated the docs to reflect it, but then I decided to change it back but forgot to revert the doc comment. Thanks for catching that 🙂